docs/tables/trello_search_organization.md

```diff
@@ -0,0 +1,9 @@
+# Table: trello_my_organization
```

Comment:
```suggestion
# Table: trello_search_organization
```

Reply:
Are we keeping this file? We don't have a table called `trello_search_organization`